vue/src/apps/ShoppingListView/ShoppingListView.vue

```diff
@@ -934,6 +934,8 @@ export default {
                 this.ui = Object.assign({}, this.ui, this.$cookies.get(SETTINGS_COOKIE_NAME))
             }
         })
+        this.$i18n.locale = window.CUSTOM_LOCALE
+        console.log(window.CUSTOM_LOCALE)
```

Comment:
Will this be in the container logs or just when developing locally?

Reply:
I’m bad about taking out console messages from debugging. That should be removed. But it would only be in the browser console.

Reply:
Haha just looked at the code bc I was curious myself :)

Reply:
Thanks for fixing the issue

Reply:
No problem. The root cause was the helper library not translating. Not sure if it was compile issue or not having access to the locale.